docs/PrairieDraw.md

```diff
@@ -214,6 +214,6 @@ These files should be committed to the `git` repository and pushed to the live s
 If you want to generate LaTeX label images without docker then you will need to install [Python](https://www.python.org), [ImageMagick](http://www.imagemagick.org/), and [LaTeX](http://tug.org/texlive/) and then run:
 
 ```sh
-cd <FULL-PATH>/PrairieLearn
+cd < FULL-PATH > /PrairieLearn
```

Comment:
I don't think it formatted this well. Is this better?

```suggestion
cd $FULL_PATH/PrairieLearn
```

Reply:
I changed this to just `cd PrairieLearn`, thoughts? Variables

Reply:
How about updating the previous line to be "...run the following command in the root of the PrairieLearn repository:"?

Reply:
Sounds great -- changed in 323bcead23a8e970244139ece0b5806c4d923908.